src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -308,6 +305,14 @@ export class MeetingManager implements AudioVideoObserver {
   audioVideoDidStop = (sessionStatus: MeetingSessionStatus): void => {
     const sessionStatusCode = sessionStatus.statusCode();
     switch (sessionStatusCode) {
+      case MeetingSessionStatusCode.MeetingEnded:
```

Comment:
How did I miss this, I thought only `AudioAttendeeEnded` was removed :(. Thanks for the fix.

Reply:
Was this supposed to replace AudioAttendeeEnded? I looked back at my change and this wasn't there before either.

Reply:
So MeetingEnded and AudioAttendeeEnded are both = 6 :) Thats why we should replace instead of removing it.

Reply:
Gotcha, I should have looked more into what it was actually doing